mkosi/config.py

```diff
@@ -1632,7 +1632,8 @@ class Config:
     passphrase: Optional[Path]
     checksum: bool
     sign: bool
-    key: Optional[str]
+    openpgp_tool: str
+    key: Optional[Union[Path, str]]
```

Comment:
We have a thing there
```suggestion
    key: Optional[PathString]
```